isthisstockgood/templates/searchbox.html

```diff
@@ -4,11 +4,11 @@
       <div class="col-md-3 nopadding">
         <input
             type="text"
+            autofocus
             class="form-control"
             id="ticker"
             placeholder="Ticker Symbol"
-            maxlength=8
-            onkeydown="return validateInput(event)"/>
```

Comment:
Appeared in the browser console as:

![cant-find-variable-validateInput](https://github.com/user-attachments/assets/1e739442-2a8c-4cf9-b019-fb45f663077c)